components/IssueCard/index.js

```diff
@@ -3,47 +3,69 @@ import { useTranslation } from 'react-i18next'
-    const { t } = useTranslation()
-    const { theme } = useTheme()
-    const isLight = theme === 'light'
+const IssueCard = ({ issue, user }) => {
+  // const { t } = useTranslation()
+  const { theme } = useTheme()
+  const isLight = theme === 'light'
 
-    return (
-        <div
-            className={`flex md:flex-row flex-col items-center justify-center gap-2 rounded-lg p-4 shadow-lg ${isLight ? 'bg-gray-200 bg-opacity-75' : 'bg-black-200 bg-opacity-75'
-                }`}
-        >
-            <div className="flex h-[40px] md:h-full w-[40px] md:w-[80px] items-center justify-center md:rounded-[20px] rounded-[10px] bg-[#99e24d]">
-                <MdGroup size={70} color="white" />
-            </div>
-            <div className="mb-4 flex w-full flex-col gap-3">
-                <div className="flex w-full items-center justify-between">
-                    <span className="md:text-[24px] text-[18px] text-[#99e24d]">{issue.title}</span>
-                    <button className="text-white mr-2 md:mr-4 rounded bg-[#99e24d] bg-opacity-30 px-2 py-1 md:text-sm text-[10px] hover:bg-[#649e26] focus:outline-none focus:ring-2 focus:ring-[#99e24d]">
-                        Apply
-                    </button>
-                </div>
-                <div className="flex w-full">
-                    <p className="text-[12px] md:text-[16px]">
-                        {issue.body}
-                    </p>
-                </div>
-                <div className="flex md:flex-row flex-col gap-3 text-gray-400">
-                    <p className="text-[16px]">
-                        <strong>Board:</strong> {issue.project_name}
-                    </p>
-                    <p className="text-[16px]">
-                        <strong>Created At:</strong> {new Date(issue.createdAt).toLocaleDateString()}
-                    </p>
-                    {issue.fields.map((field, index) => (
-                        <p key={index} className="text-[16px]">
-                            <strong>{field.field}:</strong> {field.value}
-                        </p>
-                    ))}
-                </div>
-            </div>
+  function canTakeTask(userContext, taskContext) {
+    const contextOrder = {
+      Beginner: 0,
+      Novice: 1,
+      Intermediate: 2,
+      Professional: 3,
+      Expert: 4,
+    }
+
+    const userContextValue = contextOrder[userContext]
+    const taskContextValue = contextOrder[taskContext]
+
+    return userContextValue >= taskContextValue
+  }
+
+  return (
+    <div
+      data-testid="cardIssue"
+      className={`flex flex-col items-center justify-center gap-2 rounded-lg p-4 shadow-lg md:flex-row
+        ${isLight ? 'bg-gray-200 bg-opacity-75' : 'bg-black-200 bg-opacity-75'}
+        ${
+          canTakeTask(
+            user?.contextLevel,
+            issue.fields.find((item) => item.field === 'Context Depth')?.value
+          )
+            ? 'order-0'
+            : 'order-5 opacity-50'
```

Comment:
It would be beneficial to display a message to help the user understand why they can't take the task. One idea is to change the mouse cursor to a 'Denied' icon when hovering over the task card, accompanied by a message stating, 'You don't have the required context level for this task.' Additionally, the 'Apply' button should be disabled to prevent any further action.

Reply:
what did you think?

![image](https://github.com/user-attachments/assets/f89b8573-33e4-467d-9ca2-04ad60e54d8e)